redirects-transition.conf

```diff
@@ -1620,7 +1622,7 @@ rewrite ^/pages/report_notices/* https://www.fec.gov/help-candidates-and-committ
 # pdf/ broader redirects
 rewrite ^/pdf/ar([0-9]+) https://www.fec.gov/resources/cms-content/documents/ar$1.pdf redirect;
 rewrite ^/pdf/legrec([0-9]+) https://www.fec.gov/resources/cms-content/documents/legrec$1.pdf redirect;
-rewrite ^/pdf/forms/(.*) https://www.fec.gov/resources/cms-content/documents/$1 redirect;
+rewrite ^/pdf/forms/(.*) https://www.fec.gov/resources/cms-content/policy-guidance/documents/$1 redirect;
```

Comment:
@dorothyyeager This should be `cms-content/documents/policy-guidance/`

Reply:
Fixed.